Detectors/TPC/qc/src/GPUErrorQA.cxx

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019-2025 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#define _USE_MATH_DEFINES
+
+// root includes
+#include "TFile.h"
+#include "TH1I.h"
+
+// o2 includes
+#include "TPCQC/GPUErrorQA.h"
+#include "GPUDefMacros.h"
+
+ClassImp(o2::tpc::qc::GPUErrorQA);
+
+using namespace o2::tpc::qc;
+
+//______________________________________________________________________________
+void GPUErrorQA::initializeHistograms()
+{
+  TH1::AddDirectory(false);
+
+  // get gpu error names
+  // copied from GPUErrors.h
+  static std::unordered_map<uint32_t, const char*> errorNames = {
+#define GPUCA_ERROR_CODE(num, name, ...) {num, GPUCA_M_STR(name)},
```

Comment:
Can't you just include GPUErrors.h instead of copy & pasting the code?

Reply:
Hi David, the errorNames were defined in GPUError.cxx, not in the header. I was not sure if this was done on purpose, so that's why I copied the code. 
I moved the definition now to the header, so I can include it in the QA task.